samples/bluetooth/iso_combined_bis_and_cis/README.rst

```diff
@@ -0,0 +1,125 @@
+.. _bluetooth_isochronous_combined_bis_and_cis:
+
+Bluetooth: ISO Combined BIS and CIS
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Combined BIS and CIS sample demonstrates the capability of
+a Controller to support both Broadcast Isochronous Stream (BIS) and Connected Isochronous Stream (CIS) simultaneously.
+It shows ISO-specific APIs to create a stream and to receive and send data over either CIS or BIS.
+The sample acts as a Central device that receives data over CIS and forwards the received data to a BIS.
+
+Note: The sample uses the Time of Arrival method to send data over ISO without using timestamps through a single stream (see :ref:`iso_providing_data`).
+The :ref:`bluetooth_isochronous_time_synchronization` sample can be used to learn how to synchronize data using ISO channels.
+
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+You can use any of the development kits listed above and mix different development kits.
+
+.. include:: /includes/hci_ipc_overlay.txt
+
+The sample also requires a connection to a computer with a serial terminal |ANSI| for each of the development kits.
+
+Overview
+********
+
+The sample demonstrates a device combining the CIS Central and BIS Source roles.
+As a CIS Central, it only receives data from CIS Peripheral.
+The received data is then broadcasted over a BIS.
+To provide the best performance and to minimize the number of conflicts between the CIS, BIS and ACL roles, the following measures are taken:
+* The same ISO Interval is used for both CIS and BIS. The value is set to 10 ms by default.
```

Comment:
```suggestion
The received data is then broadcasted over a BIS.

To provide the best performance and to minimize the number of conflicts between the CIS, BIS and ACL roles, the following measures are taken:

* The same ISO interval is used for both CIS and BIS.
  The value is set to 10 ms by default.
```
The only change in addition to the the linebreaks is "Interval" -> "interval", since it's not capitalized on the SDC page.